pkg/admission/annotations.go

```diff
@@ -0,0 +1,62 @@
+package admission
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/client-go/kubernetes"
+)
+
+const (
+	LastDownscaleAnnotationKey = "grafana.com/last-prepared-for-downscale"
```

Comment:
```suggestion
	LastDownscaleAnnotationKey = "grafana.com/last-downscale"
```

Reply:
@colega I just added a check for other zones having been rolled out recently. This suggestion was applied manually later.